File: Java/src/1365_HowManyNumbersAreSmallerThantheCurrentNumber/HowManyNumbersAreSmallerThantheCurrentNumber.java
class Solution {
    public int[] smallerNumbersThanCurrent(int[] nums) {
        int [] ans = new int [nums.length];
        for(int i = 0 ; i < nums.length ; i++){
            ans[i] = 0 ; 
            int currentNumber = nums[i];
            for(int j = 0 ; j < nums.length ; j++){
                if(currentNumber > nums[j]) ans[i]++;
            }
        }
        return ans;
    }
}

File: golang/1365_HowManyNumbersAreSmallerThantheCurrentNumber/howManyNumbersAreSmallerThantheCurrentNumber.go
package howmanynumbersaresmallerthanthecurrentnumber

func smallerNumbersThanCurrent(nums []int) []int {
	ans := []int{}
	for i := 0; i < len(nums); i++ {
		count := 0
		for j := 0; j < len(nums); j++ {
			if nums[i] > nums[j] {
				count++
			}
		}
		ans = append(ans, count)
	}
	return ans
}
